include/global/space.hpp

```diff
@@ -732,6 +837,7 @@ namespace Preemptive {
 				}
 				// then we check upper bounds
 				k = 0;
+				first = true;
```

Comment:
why there is no check on the jobs in the lookup table below? There should but checking that the rmax<=EST instead of rmin<=EST this time